docker/run_jakartaeetck.sh

```diff
@@ -299,10 +299,14 @@ fi
 if [ ! -z "$KEYWORDS" ];then
   if [ ! -z "$USER_KEYWORDS" ]; then
     KEYWORDS="${KEYWORDS}${USER_KEYWORDS}"
+  else
+  	KEYWORDS="${KEYWORDS}&all"
```

Comment:
Where is `all` implemented?  I see jakartaee-tck/src/com/sun/ts/lib/harness/InteropTestManager.java but am not sure if that is involved.  

I suspect that we missed removing InteropTestManager.java (sounds like it was for determining forward/reverse testing mode for interop), not sure if there is time for dealing with that.

Reply:
all is implemented here - [https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/src/com/sun/ts/lib/harness/TSHarnessObserver.java#L67](url)

Agree, we will not have time  to clean InteropTestManager, we can clean InteropTestManager later(if there is no impact on tests due to the class)

Reply:
> all is implemented here - [https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/src/com/sun/ts/lib/harness/TSHarnessObserver.java#L67](url)
> 
> Agree, we will not have time to clean InteropTestManager, we can clean InteropTestManager later(if there is no impact on tests due to the class)

What happens as a result of specifying `all` in keywords?

Reply:
keyword `all` is used to execute all test cases within testsuite. Previously(before we passed !rmiiiop&!corba) `all` used be added to keywords for default run (when no keyword was passed) here  - https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/src/com/sun/ts/lib/harness/TSHarnessObserver.java#L107 
Now we are adding same keyword at run script(used only for glassfish) to run all tests for glassfish.